demos/TransferLearning/TransferLearningDemo.py

```diff
@@ -411,86 +220,86 @@ def compile_model(self, modelName, fine=0):
-                    ax[i // 3, i % 3].set_title("Predicted:{}".format(predicted_class), color='r')
+        self.set_ngraph_bridge(backend)
 
-                    if predicted_class.lower() in self.test_generator.filenames[
-                            self.wrong_guesses[i]]:
-                        ax[i // 3, i % 3].set_title("Predicted:{}".format(predicted_class),
-                                                    color='g')
-                display(f)
+        history = self.model.fit_generator(self.train_generator,
+                                           steps_per_epoch=steps_per_epoch,
+                                           epochs=epochs,
+                                           workers=self.workers,
+                                           validation_data=self.validation_generator,
+                                           validation_steps=validation_steps,
+                                           verbose=self.verbose,
+                                           callbacks=callbacks)
+        return history
 
-        if self.verbose:
-            with self.out_stats:
-                wrong = ~(predicted_class_indices ^ self.test_class_indices) + 2
-                print("Correct Matrix")
-                print(wrong)
+    def predict(self, callbacks=None):
+        if self.model:
+            probabilities = self.model.predict_generator(self.test_generator,
+                                                         verbose=self.verbose,
+                                                         workers=self.workers,
+                                                         callbacks=callbacks)
+            predicted_class_indices = np.argmax(probabilities, axis=1)
 
-            print("Total guessed:", wrong.shape[0])
-            print("Accuracy:", np.count_nonzero(wrong) / wrong.shape[0])
+            return probabilities
 
 
 if __name__ == "__main__":
     parser = argparse.ArgumentParser(prog='TransferLearningDemo')
-    parser.add_argument('--gui', help='shows the GUI of the demo', action='store_true')
     parser.add_argument('--training',
                         help='performs the training phase of the demo',
                         action='store_true')
+    parser.add_argument('--predict',
+                        help='performs the inference phase of the demo',
+                        action='store_true')
     parser.add_argument(
         '--network_type',
         help='selects the network used for training/classification [ResNet50]/MobileNet V2')
     parser.add_argument(
-        '--backend',
-        help=
-        'selects the backend used for training/classification (run ngraph_bridge.list_backends() for full list)'
-    )
+        '--backend', help='selects the backend used for training/classification [CPU]/PLAIDML/TF]')
     parser.add_argument('--quiet', help='disables most logging', action='store_false')
+    parser.add_argument('--epochs', help='number of epochs to train')
```

Comment:
If you use `type=int` then this argument will automatically be an `int` by the time you go to use it. Plus, if a user enters something that can't be parsed as an `int`, you'll get an error.

Reply:
You can also set a default value with `default=5` here.